modules/primer-marketing-buttons/lib/button.scss

```diff
@@ -0,0 +1,32 @@
+.btn-orange { @include btn-solid($white, lighten($orange-500, 7.5%), darken($orange-500, 7.5%)); }
+.btn-blurple { @include btn-solid($white, mix($blue-400, $purple-400, 35%), mix($blue-600, $purple-600, 35%)); }
+
+.btn-outline-purple { @include btn-outline($purple); }
+.btn-outline-orange { @include btn-outline($orange); }
+.btn-outline-green { @include btn-outline($green); }
+
+@mixin btn-transparent-active {
+  color: $gray-800;
+  background-color: $white;
+  background-image: none;
+  border-color: $white;
+}
+
+.btn-transparent {
```

Comment:
This doesn't seem to be used anywhere in the `github/github` is it being used elsewhere for marketing sites?

Reply:
It is sometimes used on the ads we have on the homepage below the hero on github.com. I personally have also used it here on the hero and circular callouts https://partner.github.com/